### PR TITLE
MA-3269:Implementation to turn on log integrity feature of CloudTrail…

### PR DESCRIPTION
**Pull request by bhargavapratik**
Implementation to turn on log integrity feature of CloudTrail from UI,
Also added support for missing aws regions.

### CODE REVIEW

File: cloudtrail/index_enabler.js [hunk 1]
```diff
@@ -30,7 +30,13 @@ exports.handler = function (event, context) {
     "us-west-2": "arn:aws:iam::113285607260:root",
     "eu-central-1": "arn:aws:iam::035351147821:root",
     "ap-northeast-2": "arn:aws:iam::492519147666:root",
-    "ap-south-1": "arn:aws:iam::977081816279:root"
+    "ap-south-1": "arn:aws:iam::977081816279:root",
+    "us-east-2": "arn:aws:iam::475085895292:root",
+    "ca-central-1": "arn:aws:iam::819402241893:root",
+    "us-east-1": "arn:aws:iam::086441151436:root",
```
Comment (RajkumarShivage) [inline cloudtrail/index_enabler.js]: This is duplicate entry.

Comment (RajkumarShivage) [reply cloudtrail/index_enabler.js]: Confirm the regions once.

File: lib/aws/cloudtrail.js [hunk 2]
```diff
@@ -79,6 +79,9 @@ function AWSCloudTrail() {
     if (input.S3KeyPrefix) params.S3KeyPrefix = input.params.S3KeyPrefix;
     if (input.SnsTopicName)  params.SnsTopicName = input.params.SnsTopicName;
     if (input.multiRegion)  params.IsMultiRegionTrail = input.multiRegion;
+    if (input.LogFileValidation) params.EnableLogFileValidation = input.LogFileValidation;
```
Comment (RajkumarShivage) [inline lib/aws/cloudtrail.js]: Correct the indentation.

File: cloudtrail/index_enabler.js (hunk 1 above)
Comment (bhargavapratik) [reply cloudtrail/index_enabler.js]: Removed duplicate region

File: lib/aws/cloudtrail.js (hunk 2 above)
Comment (bhargavapratik) [reply lib/aws/cloudtrail.js]: Its correct

**Pull request merged**